src/endpoints/workers/delete_route.rs

```diff
@@ -0,0 +1,23 @@
+use super::WorkersRouteIdOnly;
+
+use crate::framework::endpoint::{Endpoint, Method};
+
+/// Delete a Route
+/// Deletes a route by route id
+/// https://api.cloudflare.com/#worker-routes-delete-route
+pub struct DeleteRoute<'a> {
+    pub zone_identifier: &'a str,
+    pub identifier: &'a str,
+}
+
+impl<'a> Endpoint<WorkersRouteIdOnly> for DeleteRoute<'a> {
+    fn method(&self) -> Method {
+        Method::Post
```

Comment:
Huh! Shouldn't the method for this be delete?

Reply:
i... think so. ok now i _know_ i didn't do manual tests on delete.